src/algorithms/SortingAlgorithms.java

```diff
@@ -20,28 +28,110 @@ public static void sortByBubbleSorting(int[] array) {
 
     public static void sortBySelectionSorting(int[] array){
 
+        for (int i = 0; i < array.length - 1; i++){
+            int minIndex = i;
+            for (int j = i + 1; j < array.length; j++){
+                if (array[j] < array[minIndex])
+                    minIndex = j;
+            }
+            if (minIndex != i)  swap(array, minIndex, i);
```

Comment:
Although  this also works, however better to always write parentheses {}, in order to avoid misreading and bugs

Reply:
Ok  thank you.